utilities/update_docs_version.sh

```diff
@@ -17,6 +17,9 @@ if [ "${RELEASED_VERSION##*-}" != "SNAPSHOT" ]; then
         sed -ri "s/:[0-9]+\.[0-9]+\.[0-9]+'/:${RELEASED_VERSION}'/g" ${item}/README.md
         sed -ri "s/\"[0-9]+\.[0-9]+\.[0-9]+\"/\"${RELEASED_VERSION}\"/g" ${item}/README.md
     done
+
+    echo "Changing version to $((RELEASED_VERSION + 1))-SNAPSHOT in gcloud-java-nio-example README"
```

Comment:
Hm, wouldn't you want the gcloud-java-nio-example README to use the released version, rather than the "-SNAPSHOT" version?  In that case, the line above could become:

```
echo "Changing version to $RELEASED_VERSION in gcloud-java-nio-example README"
```
and you could add gcloud-java-nio-examples/README.md to the list "module folders" or use a command something like this:

```
sed -ri "s/gcloud-java-nio-examples-[0-9]+\.[0-9]+\.[0-9]/gcloud-java-nio-examples-${RELEASED_VERSION}/g" gcloud-java-contrib/gcloud-java-nio-examples/README.md
```